docs/how-to-guides/running-autoware-without-cuda.md

```diff
@@ -0,0 +1,25 @@
+# Running Autoware without CUDA
+
+Although CUDA installation is recommended to achieve better performance for object detection and traffic light recognition in Autoware Universe, it is possible to run these algorithms without CUDA.
+The following subsections briefly explain how to run each algorithm in such an environment.
+
+## Running 2D/3D object detection without CUDA
+
+The following three object detection modules require CUDA:
+
+- `lidar_centerpoint`
+- `lidar_apollo_instance_segmentation`
+- `tensorrt_yolo`
+
+Instead of these modules, you can use the `euclidean_cluster` module which contains a 3D object detection algorithm that does not require CUDA. For more details, refer to the [`euclidean_cluster` module's README file](https://github.com/autowarefoundation/autoware.universe/tree/main/perception/euclidean_cluster).
```

Comment:
@kminoda I think it is useful to know the different possible configurations so I have included them in my suggestion below. Alternatively, would it be better to include the information about possible configurations in the design documentation for the Perception component?

```suggestion
Autoware Universe's object detection can be run using one of five possible configurations:

- `lidar_centerpoint`
- `lidar_apollo_instance_segmentation`
- `lidar-apollo` + `tensorrt_yolo`
- `lidar-centerpoint` + `tensorrt_yolo`
- `euclidean_cluster`

Of these five configurations, only the last one (`euclidean_cluster`) can be run without CUDA. For more details, refer to the [`euclidean_cluster` module's README file](https://github.com/autowarefoundation/autoware.universe/tree/main/perception/euclidean_cluster).
```